salt/modules/win_update.py

```diff
@@ -2,6 +2,8 @@
 '''
 Module for running windows updates.
 
+This module is being deprecated. Please use the ``win_wua`` module instead.
```

Comment:
It's probably good to add where the module will be removed here, too, so people don't have to look in the code. Maybe something like: "This module is deprecated and will be removed in Salt Fluorine. Please use..."